sdk/fd_create/prog.c

```diff
@@ -0,0 +1,80 @@
+// This test program is not ready to be released!
```

Comment:
I think this is the first small C example we have.  Can you put it under `c-examples`, in analogy to `rust-examples` that already exists in the SDK?

Reply:
Done.